packages/shell/src/cli-run.ts

```diff
@@ -40,6 +41,27 @@ export async function run(options: { env?: string; databaseURL?: string; apiKey?
     )} global variable.`
   );
 
+  const fetchApi = async (method: string, path: string, body?: any) => {
+    const { protocol, host } = parseDatabaseURL(databaseURL);
+    // TODO: Add support for staging, how?
```

Comment:
What about getting the `host` returned by `parseDatabaseURL`?

Reply:
Yeah, that could work. The CLI uses an env variable for changing the API host. Maybe that's a better approach. I'll leave the comment and let's revisit it with usage.